db/design_doc.go

```diff
@@ -90,15 +90,15 @@ const (
 	function(doc, meta) {
 	
 		//Skip any internal sync documents
-		if (meta.id.substring(0, 6) == "_sync:") {
+		if (meta.id.substring(0, 6) == "%s") {
```

Comment:
I agree with parameterising or pulling these out as constants, but the long string of repeated `base.SyncPrefix, base.SyncXattrName, base.SyncPropertyName, base.SyncXattrName, base.SyncPropertyName` in the Sprintf isn't ideal. Makes it difficult to determine where to insert new values if we need to change this.

Can we either do string concatenation inline using the constants, or maybe positional arguments? I can't decide which is better...

E.g: for inline concatenation: ```(meta.id.substring(0, 6) == "`+base.SyncPrefix+`") {```
Or Sprintf positional args https://play.golang.org/p/NMVLgVO0y0v

Reply:
Gone with positional args. I prefer the look of that.